test_bundle_image.sh

```diff
@@ -148,7 +148,19 @@ function start_container {
 		test_dir="/data/${LIFERAY_DOCKER_NETWORK_NAME}/liferay/liferay-docker/${TEST_DIR}"
 	fi
 
-	CONTAINER_ID=$(docker run -d -p 8080 -v "${test_dir}/mnt:/mnt:rw" ${network_parameters} "${LIFERAY_DOCKER_IMAGE_ID}")
+	if [[ "${LIFERAY_DOCKER_NETWORK_NAME}" == *slave* ]]
+	then
+		CONTAINER_HOSTNAME=172.17.0.1
```

Comment:
@kiwm, please check if there is a function in Docker to get this value dynamically. I'm concerned that this default value can change in the future.